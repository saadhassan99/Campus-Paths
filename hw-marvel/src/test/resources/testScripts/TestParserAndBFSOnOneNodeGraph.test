# test parser and BFS on one node graph

# build graph
LoadGraph oneNodeGraph oneNode.tsv

# check graph is built correctly
ListNodes oneNodeGraph
ListChildren oneNodeGraph onlyNode

#check BFS
FindPath oneNodeGraph onlyNode onlyNode