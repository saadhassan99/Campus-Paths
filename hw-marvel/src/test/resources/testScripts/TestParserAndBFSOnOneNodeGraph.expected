# test parser and BFS on one node graph

# build graph
loaded graph oneNodeGraph

# check graph is built correctly
oneNodeGraph contains: onlyNode
the children of onlyNode in oneNodeGraph are:

#check BFS
path from onlyNode to onlyNode:
